### DISCUSSION TITLE
Is the Ubuntu version up to date?

### DISCUSSION THREAD

**Discussion by Bustrod** [streamlink/streamlink#4046] [Q&A]
Latest Ubuntu version I can get is 1.7.0.

It works via VLC on Abema (Japanese TV station), but all other platforms return a 410 Client Error.

Any advice on what could be wrong?

**Comment by gravyboat** [comment]
Did you add the correct PPA per the [Ubuntu docs](https://streamlink.github.io/install.html#linux-and-bsd)? Looks like the latest packaged version is 2.3.0.

**Reply by Bustrod** [reply]
Thanks

Yes I followed those install instructions. I repeated and the result is: 
`streamlink is already the newest version (1.7.0+dfsg-1~webupd8~focal).`

**Reply by gravyboat** [reply]
The packager (we don't package this ourselves) isn't putting out new releases for focal it looks like: http://ppa.launchpad.net/nilarimogard/webupd8/ubuntu/pool/main/s/streamlink/

You'll have to use the [pip install instructions](https://streamlink.github.io/install.html#pypi-package-and-source-code).

**Reply by Bustrod** [reply]
That's done, updated, and I'm still on v1.7.0.

I did get this message during the pip install:
`WARNING: The script normalizer is installed in '/home/daniel/.local/bin' which is not on PATH. Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.`

I'm a Linux lay-user and don't really understand the technical details here, so I'd appreciate any further advice.

**Reply by bastimeyer** [reply]
> I did get this message during the pip install:

This is already explained in the pip install section of the docs.

Alternatively, you can just use the Linux AppImage, which is a simple standalone executable where you only have to set the executable flag (`chmod +x filename`) after downloading it.
https://streamlink.github.io/install.html#appimages

**Reply by Bustrod** [reply]
Ah it is too.

This is beyond me and I'll probably break something. I'll watch in browser, as god intended :D
